etc/profile-m-z/xlinks2

```diff
@@ -0,0 +1,21 @@
+# Firejail profile for xlinks2
+# Description: Text WWW browser (X11)
+# This file is overwritten after every install/update
+# Persistent local customizations
+include xlinks2.local
+# Persistent global definitions
+# added by included profile
+#include globals.local
+
+noblacklist /tmp/.X11-unix
+noblacklist ${HOME}/.links
```

Comment:
links or links2?

Reply:
actually it was an error in xlinks.profile